examples/website/textures/app.js

```diff
@@ -30,6 +31,10 @@ void main() {
 }
 `;
 
+const Header = styled.h2`
```

Comment:
This is certainly OK but perhaps ask if it is worth adding a new dependency just for this? examples are supposed to be fairly minimal - idea is that users can fork this code and use it as basis for their own code.

Reply:
Ok, will be removed in next PR